client/post-editor/editor-ground-control/index.jsx

```diff
@@ -283,6 +286,12 @@ export class EditorGroundControl extends PureComponent {
 						</span>
 					}
 				</div>
+				{ hasRevisions &&
```

Comment:
Don't we need to use a `isEnabled( 'post-editor/revisions' )` guard here as this is not yet released?